README.md

```diff
@@ -7,7 +7,7 @@
 ## Introduction
 
 This plugin allows pipelines to use standard Jenkins SCM plugins to check out source code.
-The goals are the maximum possible compatibility with existing plugins, and great flexibility for script authors.
+The goals are the maximum possible compatibility with existing plugins, and great flexibility for script authors. The ["Pipeline Syntax" snippet generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) guides the user to define the checkout step.
```

Comment:
Thanks!  We prefer sentence per line formatting so that changes inside a sentence are easier to review.
```suggestion
The goals are the maximum possible compatibility with existing plugins, and great flexibility for script authors.
The ["Pipeline Syntax" snippet generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) guides the user to define the checkout step.
```

Reply:
Thank you for feedback @MarkEWaite. I made another commit to put the sentence on a newline.